nautobot/core/api/serializers.py

```diff
@@ -122,6 +122,7 @@ class BaseModelSerializer(OptInFieldsMixin, serializers.HyperlinkedModelSerializ
     display = serializers.SerializerMethodField(read_only=True, help_text="Human friendly display value")
     object_type = ObjectTypeField()
     composite_key = serializers.SerializerMethodField()
+    natural_slug = serializers.SerializerMethodField()
```

Comment:
Can/should we have a test in the generic API test case that checks to make sure `natural_slug` is present and correctly populated on all serializers? Maybe just an extra check in `APIViewTestCases.GetObjectViewTestCase.test_get_object`?